cypress/support/pages/contactForm.pageObject.js

```diff
@@ -1,38 +1,69 @@
+/// <reference types='cypress' />
 import PageObject from '../PageObject';
 
 class ContactFormPageObject extends PageObject {
-  url = '/index.html';
+  get nameField() {
+    return cy.get('#name');
+  }
 
-  get emailField() {
-    return cy.get('#recipient-email');
+  get countryField() {
+    return cy.get('#country');
   }
 
-  get nameField() {
-    return cy.get('#recipient-name');
+  get cityField() {
+    return cy.get('#city');
+  }
+
+  get creditCartField() {
+    return cy.get('#card');
   }
 
-  get messageField() {
-    return cy.get('#message-text');
+  get monthField() {
+    return cy.get('#month');
   }
 
-  get sendMessageBtn() {
-    return cy.contains('.btn', 'Send message');
+  get yearField() {
+    return cy.get('#year');
   }
 
-  typeEmail(email) {
-    this.emailField.type(email, { force: true });
+  typeCountry(country) {
+    this.countryField.type(country);
   }
 
   typeName(name) {
-    this.nameField.type(name, { force: true });
+    this.nameField.type(name);
+  }
+
+  typeCity(city) {
+    this.cityField.type(city);
+  }
+
+  typeCreditCart(card) {
+    this.creditCartField.type(card);
+  }
+
+  typeMonth(month) {
+    this.monthField.type(month);
+  }
+
+  typeYear(year) {
+    this.yearField.type(year);
+  }
+
+  clickBtn() {
+    this.Btn.click();
   }
 
-  typeMessage(message) {
-    this.messageField.type(message);
+  clickOnTheButton(name) {
+    cy.contains('.btn', name)
+      .click();
   }
 
-  clickOnSendMessageBtn() {
-    this.sendMessageBtn.click();
+  assertData(name, card) {
+    cy.get('.sweet-alert')
```

Comment:
In the 'assertData' method, you should also check for the other fields (country, city, card, month, year) to ensure they match the expected values.